.env

```diff
@@ -0,0 +1,2 @@
+MONGOHQ_URL=mongodb://heroku:c-rJompa5mbNDiERpkBJMfKXbfSazY_YaBGqrnFzmwgao-WRaPHaU_ZED_TU7zp2C8lp3OV1azqFINmN6EQ27A@candidate.37.mongolayer.com:10928,candidate.13.mongolayer.com:11150/app40069500
```

Comment:
Good catch, this was my fault.  I removed the files from the history, changed the db password, and made sure that the .gitignore is set up correctly.  Thanks for catching this!